files/build_templates/per_namespace/lldp.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
+# This delay is for fast-reboot performance
```

Comment:
Correction - fast-reboot and warm-reboot performance.

Reply:
Done